apps/site/assets/js/algolia-result.js

```diff
@@ -537,7 +527,7 @@ export function parseResult(hit, index) {
       index === "projects" ||
       null,
     hitFeatureIcons: getFeatureIcons(hit, index),
-    id: hit.place_id || null
+    id: hit.id || null,
```

Comment:
The `id` is used in the template above to render the link, and then the code that hooks up the click listeners look for that `id`. It's not ideal :/ but refactoring the code there seems nontrivial. I think this could be something we save for after the initial release but it's up to you.